File: slurm/sleep.R
args <- commandArgs(trailingOnly = TRUE)
t <- as.double(args)
cat("the time is now ",as.character(Sys.time()),'\n')
cat("zzzzzz",'\n')
Sys.sleep(t)
cat("hello - I slept for ",t," seconds",'\n')
cat("the time is now ",as.character(Sys.time()),'\n')

File: slurm/sleep.py
import sys
import time
import datetime

print(sys.version)
t = int(sys.argv[1])
print("the time is now ",datetime.datetime.now())
print("zzzzzz")
time.sleep(t)
print("hello - I slept for ",t," seconds")
print("the time is now ",datetime.datetime.now())
